### PR TITLE
fonts, favicon, colors

### PR DESCRIPTION
**Pull request by designmatty**
### Changes
- updates font
- Transit Center to TransitCenter (removed space)
- purple to blue
- adds favicon and social meta tags in head

### Demo
screen shot 2017-11-08 at 12 58 46 pm

### CODE REVIEW

File: app/assets/sass/vendors/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+This CSS resource incorporates links to font software which is the valuable copyrighted property of Monotype and/or its suppliers. You may not attempt to copy, install, redistribute, convert, modify or reverse engineer this font software. Please contact Monotype with any questions regarding Web fonts: http://www.fontshop.com
```
Comment (CloudNiner) [inline app/assets/sass/vendors/_fonts.scss]: That's some harsh language right there...

Comment (CloudNiner) [reply app/assets/sass/vendors/_fonts.scss]: We have the license to use this?

Comment (designmatty) [reply app/assets/sass/vendors/_fonts.scss]: This is what the TransitCenter sent directly to us when we asked them for access to their fonts.

Comment (designmatty) [reply app/assets/sass/vendors/_fonts.scss]: Better yet, I'm going to update this. I'll pull from where they host their font.

Comment (CloudNiner) [reply app/assets/sass/vendors/_fonts.scss]: Cool. 👍

**Pull request merged**